### ISSUE TITLE
Parameter tuning and re-implementation with Pytorch

### ISSUE THREAD

**Issue by htzheng** [zsyzzsoft/co-mod-gan#19]
First, thank you for the impressive work! Currently, I am re-implementing a pytorch version of co-mod-gan, and I have several questions regarding the model:

1. Have you tried different R1 regularization? Empirically, I found that when using a small R1 than 10, the convergence of l1 loss is faster, I wonder if you tried other R1 weights?
2. Will dropout of the global code improves the performance?
3. Have you tried adding a skip connection to the encoder?
4. Also why the style mixing weight is set to 0.5?

Thanks

**Comment by zsyzzsoft** [comment]
Unfortunately I may not have useful information regarding your questions. Most of the hyperparameters were only chosen by intuition as we didn't have much resource to run the experiments.

**Comment by styler00dollar** [comment]
This sounds amazing @htzheng. I am looking forward for the code. Good luck.

**Comment by htzheng** [comment]
@styler00dollar It is still hard for me to release the code while I am doing the summer internship, but I will try releasing the code after September. You could try modifying the training code and model from https://github.com/rosinality/stylegan2-pytorch
